jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -550,7 +552,7 @@ private void pushImages() {
               Verify.verifyNotNull(results.baseImagesAndLayerPushResults.get().get(baseImage)));
 
           Future<BlobDescriptor> containerConfigPushResult =
-              results.builtImagesAndContainerConfigurationPushResults.get().get(builtImage);
+              results.baseImagesAndContainerConfigPushResults.get().get(baseImage);
```

Comment:
was this previously wrong?

Reply:
No. Previously the key of the map was built images, so `Map.get()` accepted `builtImage`. Now this PR changed the key to base images, so `Map.get()` should accept `baseImage`. The end result remains same.